packages/svelte/src/reactivity/map.test.ts

```diff
@@ -192,14 +192,14 @@ test('map handling of undefined values', () => {
 		render_effect(() => {
 			log.push(map.get(1));
 		});
+	});
 
-		flushSync(() => {
-			map.delete(1);
-		});
+	flushSync(() => {
+		map.delete(1);
+	});
```

Comment:
what's the purpose of these changes? they pass on `main` already, so what are they testing exactly?

Reply:
That's because `effect_root` didn't create a `branch` before. Because child effects get attached to their parent _after_ their contents run, the `flushSync` doesn't do anything as the root has no child effects.